src/dms/core/EdFi.DataManagementService.Core/ApiSchema/ApiSchemaFileLoader.cs

```diff
@@ -118,15 +124,48 @@ private static List<JsonNode> ReadApiSchemaFiles(string directoryPath)
         }
         else
         {
-            Assembly assembly =
-                Assembly.GetAssembly(typeof(DataStandard52.ApiSchema.Marker))
-                ?? throw new InvalidOperationException("Could not load assembly-bundled ApiSchema file");
+            string baseDirectory = AppDomain.CurrentDomain.BaseDirectory;
+
+            string coreAssemblyPath = Path.Combine(baseDirectory, "EdFi.DataStandard52.ApiSchema.Core.dll");
```

Comment:
We should avoid hardcoding data standard package name to allow different DS versions. API should support importing more than one extension package.

Reply:
I think in future , else part code is not needed once we started using  ApiSchemaDownloader utility to bring them into folder /EdFi.DataStandard52.ApiSchema/